website/docs/d/datasource_compute_region_instance_group.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "google"
+page_title: "Google: google_compute_instance_group"
+sidebar_current: "docs-google-datasource-compute-instance-group"
+description: |-
+  Get a Compute Region Instance Group within GCE.
```

Comment:
The big feature is being able to list the instances inside the instance group. We should make that clear here.

Reply:
Done!  Took a lot of tiny commits to test the website - sorry about the mess, of course I'll squash before I merge.